### ISSUE TITLE
Actualize this

### ISSUE THREAD

**Issue by olegakbarov** [GetExpert/redux-blog-example#32]
How about to bring back good ol' days and refresh the repo @faergeek ?

**Comment by runtheroad** [comment]
PvP ili zassal?

**Comment by faergeek** [comment]
What do you want to refresh? Upgrade packages or something else?

**Comment by olegakbarov** [comment]
Yeah, packages are good start. But we can do better — have you learned anything about building of SPA's lately?

**Comment by drudv** [comment]
I upgraded packages and did some corresponding fixes here: https://github.com/drudv/react-redux-starter-kit 
If you guys still plan to actualize it, I would make a pull-request with the changes.

**Comment by olegakbarov** [comment]
@drudv yeah, I glanced at your commit: PR is welcomed!

**Comment by drudv** [comment]
Great. PR is created, please review: https://github.com/olegakbarov/react-redux-starter-kit/pull/39
